client/src/App.jsx

```diff
@@ -2,11 +2,14 @@ import React from 'react';
 import { BrowserRouter, Route } from 'react-router-dom';
 import { createBrowserHistory } from 'history';
 import MainPage from './components/MainPage/MainPage';
+import GA from './common/GoogleAnalytics';
+
 
 export const history = createBrowserHistory();
 
 const App = () => (
   <BrowserRouter>
+    { GA.init() && <GA.RouteTracker /> }
```

Comment:
From what I can tell this is only able to track visits to each of the pages. What else does it do? not totally clear from the code